guide/creating-your-bot/command-handling.md

```diff
@@ -111,6 +111,50 @@ client.on(Events.InteractionCreate, async interaction => {
 });
 ```
 
+## Command categories
```

Comment:
Should this be integrated with subcommands so that categories are commands and their files are subcommands and make this actually useful